src/AspNetCoreModuleV2/AspNetCore/src/applicationinfo.cpp

```diff
@@ -178,13 +172,13 @@ APPLICATION_INFO::UpdateAppOfflineFileHandle()
 
 HRESULT
 APPLICATION_INFO::EnsureApplicationCreated(
-    IHttpContext *pHttpContext,
-    STRU* struExeLocation
+    IHttpContext *pHttpContext
 )
 {
     HRESULT             hr = S_OK;
     BOOL                fLocked = FALSE;
     IAPPLICATION       *pApplication = NULL;
+    STRU                exeLocation;
```

Comment:
please follow the same naming conversation and rename it to struExeLocation as it helps to identify the type of this variable